README.md

```diff
@@ -1,72 +1,100 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Filmer
 
-> Description the project.
+> Hello!
+Filmer is a website that gives you information about the latest films and shows. Re-creating this website you will see how to give some functionality to your sections and make your website more user-interactive. Also you will see how to use callbacks and promises, proper ES6 syntax, webpack to bundle JavaScript, jest and API.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML
+- CSS
+- JavaScript
+- Webpack
+- Linters
+- Jest
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+Setting up this project locally:
+#### On your terminal:
+- Clone this repository to your computer: git clone git@github.com:emiliazm/leaderboard.git.
+- Open the project on your code editor: "cd Todo-list" and "code ."
+- Install npm package: npm install.
+- Run website: npm run start.
 
-To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
+## Live Demo (if available)
 
-### Install
+[Live Demo Link] (https://emiliazm.github.io/webapp/)
```

Comment:
[OPTIONAL]
- I think the closing square bracket and opening bracket should be placed close to each other so the link could look better. Currently, it looks like this 
![image](https://user-images.githubusercontent.com/45460192/169415096-fc507ea2-133e-47a2-8148-003eea352824.png)